plugins/@grouparoo/sqlite/__tests__/utils/data.ts

```diff
@@ -0,0 +1,142 @@
+import { connect } from "../../src/lib/connect";
+import path from "path";
+import fs from "fs";
+import os from "os";
+import parse from "csv-parse/lib/sync";
+import { config } from "actionhero";
+
+const workerId = process.env.JEST_WORKER_ID || 1;
+export const usersTableName = `users_${workerId}`;
+export const purchasesTableName = `purchases_${workerId}`;
+export const profilesDestinationTableName = `output_users_${workerId}`;
+export const groupsDestinationTableName = `output_groups_${workerId}`;
+
+const allTables = {
+  [usersTableName]: `
+CREATE TABLE ${usersTableName} (
+  "id" integer PRIMARY KEY,
+  "first_name" text,
+  "last_name" text,
+  "email" text,
+  "gender" text,
+  "ip_address" text,
+  "ios_app" boolean,
+  "android_app" boolean,
+  "vip" boolean,
+  "ltv" float,
+  "date" date,
+  "stamp" datetime
+)
+`,
+  [purchasesTableName]: `
+CREATE TABLE ${purchasesTableName} (
+  "id" integer PRIMARY KEY,
+  "profile_id" integer,
+  "purchase" text,
+  "amount" float,
+  "date" date,
+  "stamp" datetime
+)
+`,
+  [profilesDestinationTableName]: `
+CREATE TABLE ${profilesDestinationTableName} (
+  "id" integer PRIMARY KEY,
+  "customer_email" text,
+  "fname" text,
+  "lname" text,
+  UNIQUE(customer_email)
+)
+`,
+  [groupsDestinationTableName]: `
+CREATE TABLE ${groupsDestinationTableName}(
+  "id" integer PRIMARY KEY,
+  "userId" integer NOT NULL,
+  "group" text NOT NULL,
+  UNIQUE("userId", "group")
+)
+`,
+};
+
+export const appOptions = {
+  file: path.join(os.tmpdir(), `${config.sequelize.database}.sqlite`),
+};
+
+const appId = "app_31bb06e8-0a4e-49c3-ad42-545f2e8662e1";
```

Comment:
FYI this can be anything, like `test_app` in the future

Reply:
I made the change anyways, just to simplify.